requirements.in

```diff
@@ -20,12 +20,13 @@ factory-boy
 fakeredis
 freezegun
 https://github.com/codecov/opentelem-python/archive/refs/tags/v0.0.4a1.tar.gz#egg=codecovopentelem
-https://github.com/codecov/shared/archive/9c31870de026a8a3f026f0753b1991e8c832fadb.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/f96b72f47583a083e0c78ec7c7ff4da5d3ad6a19.tar.gz#egg=shared
```

Comment:
Would this also need to get rid of the things in `services/task.py`?

Reply:
Yes. We should probably still upgrade that at some point, but this only loosens the allowable dependencies to let us use an older version of `sentry-sdk`.